kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -232,8 +232,8 @@
         TaskResource.get(this.syncTaskId).then(task => {
           if (task.clearable) {
             this.isSyncing = false;
-            this.syncTaskId = '';
             TaskResource.clear(this.syncTaskId);
+            this.syncTaskId = '';
```

Comment:
Why did this need to be moved? The clear method returns a promise, so if we want to wait until after the clear we would need to await it.

Reply:
Oh no no, I am not sure the reason for setting an empty string to `syncTaskId`, but the thing was that if we first set `this.syncTaskId = '';` and after we execute `TaskResource.clear(this.syncTaskId);`, we would be excecuting TaskResource.clear with an empty string, and because of this, it was throwing an exception, and the code below wasn't executing, and therefore the last sync value wasn't being updated.

Reply:
Ah, that makes a lot of sense, yes!